src/main/java/menion/android/whereyougo/gui/extension/activity/CustomMainActivity.java

```diff
@@ -169,12 +167,10 @@ public boolean dispatchKeyEvent(KeyEvent event) {
                     CustomMainActivity.this.finish();
                     return true;
                 case CLOSE_DESTROY_APP_DIALOG_NO_TEXT:
-                    showDialogFinish(FINISH_EXIT);
-                    return true;
                 case CLOSE_DESTROY_APP_DIALOG_ADDITIONAL_TEXT:
                     showDialogFinish(FINISH_EXIT);
                     return true;
-                case CLOSE_HIDE_APP:
+                default:
                     // no action
```

Comment:
Please let the `                case CLOSE_HIDE_APP:`. So it is clearer when reading the code that this possibility was not overseen. The `default:` can stay anyway.

Reply:
Added in the next force push